style.css

```diff
@@ -100,5 +99,95 @@ nav{
 	font-size: 12px;
 	color: black;
 	display: inline-block;
-	margin-top: 15px;
+	margin-top: 15px;	
+}
+/*Section*/
+.content {
+	margin-left: auto;
+	margin-right: auto;
+	width: 900px;
+}
+header{
+	height: 60px;
+	border-bottom: 1px solid #ccc;
+}
+h1{
+	font-size: 40px;
+	color: black;
+	padding-top: 15px;
+	padding-bottom: 15px;
+}
+article{
+	height: 600px;
+	float: left;
+	width: 400px;
+}
+aside{
+	height: 600px;
+	float: right;
+	width: 500px;
+}
+.image{
+	padding: 80px 0px 100px 20px;
+}
+.sign-up{
+	
+	height: 150px;
+}
+.sign-up h1{
+	font-weight: bold;
+	color: black;
+	font-size: 24px;
+}
+.sign-up p{
+	font-size: 14px;
+	color: #505050;
+}
+.email {
+	height: 80px;
+}
+
+.email h2{
+	background: white;
+	padding-bottom: 5px;
+	font-weight: bold;
+	font-size: 16px;
+	font-family: sans-serif;
+	height: 25px;
+}
+.email form{
+	height: 50px;
+}
+
+input{
+	height: 40px;
+	width: 370px;
+	background: #e4e4e4;
+	padding-left: 15px;
+	font-weight: bold;
+	font-family: sans-serif;
+	font-size: 12px;
+}
```

Comment:
you need to specify `border: none` to over-write the browser's inbuilt styles for `input`.